src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
@hellobrian Thank you for asking. It’s for generating breadcrumb items from data. I’ll send an example in our internal repo.